auth-lib/build.gradle

```diff
@@ -41,9 +41,6 @@ android {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android.txt')
         }
-        debug {
-            testCoverageEnabled true
```

Comment:
This makes the sample-app crash. I can't understand why but removing this fixes it